src/Actions/NotifyUsersWorkflowAction.php

```diff
@@ -164,7 +164,7 @@ public function getContextFields(DataObject $target)
             return $result;
         }
 
-        $fields = $target->db();
+        $fields = $target->getSchema()->fieldSpecs($target);
```

Comment:
Any reason for choosing this method over say `DataObjectSchema::databaseFields()`?

Reply:
1). Not really but (see 2)
2). Seeing as $target already has a schema accessible, seems wasteful to re-instantiate it statically
3). I was looking for something like:

```
$fields = $target->databaseFields(); // <-- No need to pass anything as an arg here
```

Reply:
Aka `$target->db()` in SS3 :-P

Reply:
Sorry, the changelog is misleading. This is the suggested drop in replacement =)